broke-sample/src/main/java/org/dsngroup/broke/SamplePublish.java

```diff
@@ -36,6 +36,13 @@ public static void main(String[] args) {
             BlockClient blockClient = new BlockClient(args[0], Integer.parseInt(args[1]));
 
             blockClient.connect(0, 0);
+
+            // Stress test
+            // for(int i=0; i<65535; i++) {
+            //    logger.info("Publish "+i);
+            //    blockClient.publish("Foo", MqttQoS.AT_LEAST_ONCE, 0, "Bar");
+            // }
+
             while(System.in.read()!=0) {
                 blockClient.publish("Foo", MqttQoS.AT_LEAST_ONCE, 0, "Bar");
             }
```

Comment:
Actually, the performance bottleneck I think, is not the **single client publish side**. That is, to maximized a single client's frequent publish, is not a real scenario. This is the why the implementation is wrong in previous pr. 
You should figure out the multiple clients scenario, as assigned multiple threads for them to publish and subscribe.
i.e.
```java
Thread[] publisherThreads = new Thread[10]
Thread[] subscriberThreads = new Thread[100]
```

Additionally, to test the rtt, you can write a test program for **sleep and response** to simulate the variational rtt.

Reply:
I see, this test is for the packet identifier assignment. All publish packets should have packet identifier between 1 and 65535.